### DISCUSSION TITLE
Since updating to 0.0.5 sync is not staring and blocks just spin

### DISCUSSION THREAD

**Discussion by zinc335** [Chia-Network/chia-blockchain#7341] [Support]
Any suggestions...
![Capture](https://user-images.githubusercontent.com/87233479/125164498-6c153e00-e18a-11eb-98b4-d0950f2456b5.JPG)

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
